test/cutting/test_wire_cutting_transforms.py

```diff
@@ -205,9 +207,9 @@ def resulting_circuit4() -> tuple[QuantumCircuit, list[int]]:
         ("circuit4", "resulting_circuit4"),
     ],
 )
-def test_transform_cuts_to_moves(request, sample_circuit, resulting_circuit):
+def test__transform_cuts_to_moves(request, sample_circuit, resulting_circuit):
```

Comment:
what exactly is the extra underscore here?

Reply:
Ha!  This was not intended.  It came about when I did a find-and-replace to add a leading underscore to `transform_cuts_to_moves`, once I realized that only `cut_wires` is immediately useful.  Maybe we'll find a use for `_transform_cuts_to_moves` one day, in which case I expect we'll make it public.  If not, we might later decide to refactor the tests not to depend on it.

Reply:
I thought you were testing whether I was actually reviewing all your PRs or not lol ;)